src/codemod.py

```diff
@@ -80,6 +83,15 @@
 
 
 import sys, os
+def is_extensionless(path):
+  """
+  Returns True if path has no extension.
+  """
```

Comment:
It'd be great to have some doctests here, since those are the only kinds of tests in codemod at the moment.